lib/CXGN/Genotype/Search.pm

```diff
@@ -2271,24 +2267,20 @@ sub get_pcr_genotype_info {
 
 }
 
-sub get_protocol_id_from_genotype_id {
```

Comment:
You modified this function including function name, but the "get_protocol_id_from_genotype_id" is being used in line no. 1621